webservices/config.py

```diff
@@ -19,14 +19,15 @@ def get_cycle_end(year):
     return year if year % 2 == 0 else year + 1
 
 CURRENT_YEAR = datetime.datetime.now().year
+CURRENT_CYCLE = datetime.datetime.now().year + datetime.datetime.now().year % 2
```

Comment:
Same question as before - given the cycle methods defined above, do we need this?  And if so, should it be defined only here and not in the other spot too to cut down on duplication?